<launch>
    <node pkg="turtlesim" type="turtlesim_node" name="sim"/>
    <node pkg="turtlesim" type="turtle_teleop_key" name="teleop" output="screen"/>
    <node name="command_publisher" pkg="dynamic_tutorials" type="random_controller.py" output="screen" clear_params="true" />
</launch>
